### DISCUSSION TITLE
.MTS Video playback takes a long time for initial buffer/load

### DISCUSSION THREAD

**Discussion by jvanvolken** [photoprism/photoprism#1385] [Q&A] [answered]
Howdy PhotoPrism team and community! 

Videos with the .MTS extension take a very long time to play initially. Once loaded, they buffer just fine. 

Any ideas?

**Comment by lastzero** [comment, marked as answer]
They get transcoded on demand, so it may take some time - depending on the CPU, I/O performance and video size. You may run the `photoprism convert` command in a terminal to transcode all files upfront.

**Reply by jvanvolken** [reply]
That sounds about right, haha!

Thanks for the info!
